src/renderer/lib/state.js

```diff
@@ -207,6 +207,8 @@ function load (cb) {
         onSavedState(err)
         return
       }
+    } else if (saved.prefs.externalPlayerPath == null) {
+      saved.prefs.externalPlayerPath = ''
```

Comment:
The correct way to handle old configuration values is by adding an entry to the `migrations.js` file to "upgrade" the user's config when they update to a new version of the app. This way, we can handle the migration in one place. Then, throughout the rest of the codebase we can just assume that the configuration file has the new format. So, in this case, we could just assume that `saved.prefs.externalPlayerPath` will always be a string going forward :)

Reply:
I added the migration, assuming the next version will be 0.21.1.